server/src/typeDefs/subscriptionPlan.ts

```diff
@@ -0,0 +1,23 @@
+import { gql } from 'apollo-server'
+
+export const subscriptionPlan = gql`
+  scalar DateTime
+  type SubscriptionPlan {
+    id: String!
+    type: String!
+    createdAt: DateTime!
+    expirationDate: DateTime!
+    originalExpirationDate: DateTime!
+    user: User!
+    userId: String!
+  }
+
+  type AvailableSubscriptionPlan {
```

Comment:
i feel like these two types can just be combined. also without any comments it's basically impossible to understand what the difference is.